README.md

```diff
@@ -66,13 +68,15 @@ Or update submodules at any time with:
 git submodule update --init --recursive
 ```
 
+NOTE: The previous steps does not apply to Windows users, because no command-line method allow user to clone/get a github repository.
+
 Then, to build a single jar:
 
 ```bash
 ./gradlew fatjar
 ```
 
-The Simplify jar will be in `simplify/build/libs/simplify.jar`. You can test it's working by simplifying the obfuscated-example app:
+The Simplify jar will be in ``simplify/build/libs/``. You can test it's working by simplifying the obfuscated-example app. Here's how you'd run it (you may need to rename the output simplify jar file to simplify.jar if the name do not match):
```

Comment:
do not -> doesn't

Reply:
Oops... Thought I didn't make any mistake...